### PR TITLE
[Zaraz] Cleanup

### PR DESCRIPTION
**Pull request by marciocloudflare**
- Cleaned out of date steps
- Added new Edit pages to improve readability
- Removed backticks from properties table
- Addresses [PCX-3351](https://jira.cfops.it/browse/PCX-3351), [PCX-3352](https://jira.cfops.it/browse/PCX-3352) and [PCX-3356](https://jira.cfops.it/browse/PCX-3356)

### CODE REVIEW

File: products/zaraz/src/content/get-started/add-tool.md [hunk 1]
```diff
@@ -15,18 +15,18 @@ You can add new third-party tools and load them into your website through the Cl
 
 1. Choose a tool from the tools library page and click **Continue** to confirm your selection.
 
-1. In **Set up**, configure the settings for your new tool. The information you need to enter will depend on the tool you choose.
+1. In **Set up**, configure the settings for your new tool. The information you need to enter will depend on the tool you choose. 
+
+1. Optional: Click **Add field** to add additional fields to your tool. Refer to [Custom fields](/get-started/custom-fields) for more information.
```
Comment (pedrosousa) [inline products/zaraz/src/content/get-started/add-tool.md]:
```suggestion
1. (Optional) Click **Add field** to add extra fields to your tool. Refer to [Custom fields](/get-started/custom-fields) for more information.
```
Avoiding `add additional`.
Also, this is the formatting I used recently for optional steps... Maybe we can discuss this internally next week?

File: products/zaraz/src/content/get-started/add-tool.md [hunk 2]
```diff
@@ -15,18 +15,18 @@ You can add new third-party tools and load them into your website through the Cl
...
 
 1. Click **Save**.
 
-Your tool is now configured. Cloudflare Zaraz automatically creates the `Pageview` event that will load the tool into your website when the conditions you set are met. 
+Your tool is now configured. Depending on the tool you chose, Cloudflare Zaraz might automatically create a `Pageview` event. Refer to [Send events to a third-party tool](/get-started/send-events) to learn more.
 
-From this page you can configure the tool's **Settings** or add additional events by clicking **Create event**. You can also edit the events listed by clicking their names.
+On this page you can configure the tool's **Settings**, add additional events by clicking **Create event**, and click the events listed to edit them.
```
Comment (pedrosousa) [inline products/zaraz/src/content/get-started/add-tool.md]:
```suggestion
On this page you can configure the tool's **Settings**, add extra events by clicking **Create event**, and edit events by clicking on them.
```
I'd mention the operation first (edit) and then the step (click).

File: products/zaraz/src/content/get-started/edit-tools-and-events.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+pcx-content-type: how-to
+---
+
+# Edit tools and events
+
+1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com/login), and select your account and website. 
+1. Go to **Zaraz**.
+1. Under **Third-party tools**, locate your tool and click **Edit**.
+
+On this page you will be able to edit settings related to the tool, add additional events, and edit existing ones. To edit an existing event, click its name.
```
Comment (pedrosousa) [inline products/zaraz/src/content/get-started/edit-tools-and-events.md]:
```suggestion
On this page you will be able to edit settings related to the tool, add extra events, and edit existing ones. To edit an existing event, click its name.
```

File: products/zaraz/src/content/get-started/send-events.md [hunk 4]
```diff
@@ -20,10 +20,10 @@ To start using events, first create a trigger to determine when this bundle of i
...
+1. In the **Firing Triggers** field, choose the relevant trigger or triggers you [previously created](/get-started/create-trigger). If you choose more than one trigger, the event will only be sent to the third-party tool when all the selected triggers are matched.
 
-1. Depending on the tool, you may also choose an **Event type** — either **Pageview** or **Event**. The options you can fill out in **Event** will depend on the tool you are configuring.
+1. Depending on the tool you are adding, you might also have the option to choose an **Event type**.
 
 1. Click **Save**.
 
-The new event you created will appear in the third-party tool, in the **Events** tab, under **Tracking Events**. 
\ No newline at end of file
+The new event will appear under **Tracking Events**. To edit an event, refer to [Edit tools and events](/get-started/edit-tools-and-events)
```
Comment (pedrosousa) [inline products/zaraz/src/content/get-started/send-events.md]:
```suggestion
The new event will appear under **Tracking Events**. To edit an event, refer to [Edit tools and events](/get-started/edit-tools-and-events).
```

**Review by pedrosousa — APPROVED** [review]

File: products/zaraz/src/content/get-started/add-tool.md (hunk 1 above)
Comment (marciocloudflare) [reply products/zaraz/src/content/get-started/add-tool.md]: Sure. I wasn't totally sure of the formatting for optional steps, but ended up going with what we have in our content type guidelines

**Pull request merged**